docs/docs/cmd/teams/message/message-restore.mdx

```diff
@@ -0,0 +1,60 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams message restore
+
+Restores a deleted message from a channel in a Microsoft Teams team
+
+## Usage
+
+```sh
+m365 teams message restore [options]
+```
+
+## Options
+
+```md definition-list
+`--teamId [teamId]`
+: ID of the Microsoft Teams team. Specify either `teamId` or `teamName` but not both.
+
+`--teamName [teamName]`
+:	Name of the Microsoft Teams team. Specify either `teamId` or `teamName` but not both.
+
+`--channelId [channelId]`
+:	Channel ID of the Microsoft Teams team. Specify either `channelId` or `channelName` but not both.
+
+`--channelName [channelName]`
+:	Channel name of the Microsoft Teams team. Specify either `channelId` or `channelName` but not both.
+
+`-i, --id <id>`
+:	The ID of the Teams message.
+```
+
+<Global />
+
+## Remarks
+
+You can only restore Microsoft Teams messages that you created yourself.
```

Comment:
Let's put this behind the admonition.

Reply:
I don't want to be difficult but remarks come with other commands before the info block. You sure you want the remark after the info block?

Reply:
Or you just want line 42 inside the info block? Bit confused here 😅

Reply:
> I don't want to be difficult but remarks come with other commands before the info block. You sure you want the remark after the info block?

Depends, maybe for older commands? AFAIK, when creating new commands, most of the time we put the admonitions in front of other remarks.

> Or you just want line 42 inside the info block? Bit confused here 😅

Yes, the message about the delegated permissions should be included in the admonition. Additionally, I'd place `You can only restore Microsoft Teams messages that you created yourself.` below it.